services/cd-service/pkg/cmd/server.go

```diff
@@ -58,48 +58,47 @@ const (
-	MaximumQueueSize         uint          `default:"5" split_words:"true"`
-	AllowLongAppNames        bool          `default:"false" split_words:"true"`
-	ArgoCdGenerateFiles      bool          `default:"true" split_words:"true"`
-	DbOption                 string        `default:"NO_DB" split_words:"true"`
-	DbLocation               string        `default:"/kp/database" split_words:"true"`
-	DbCloudSqlInstance       string        `default:"" split_words:"true"`
-	DbName                   string        `default:"" split_words:"true"`
-	DbUserName               string        `default:"" split_words:"true"`
-	DbUserPassword           string        `default:"" split_words:"true"`
-	DbAuthProxyPort          string        `default:"5432" split_words:"true"`
-	DbMigrationsLocation     string        `default:"" split_words:"true"`
-	DexDefaultRoleEnabled    bool          `default:"false" split_words:"true"`
-	DbWriteEslTableOnly      bool          `default:"false" split_words:"true"`
-	ReleaseVersionsLimit     uint          `default:"20" split_words:"true"`
-	DeploymentType           string        `default:"k8s" split_words:"true"` // either k8s or cloudrun
-	CloudRunServer           string        `default:"" split_words:"true"`
+	GitUrl                string        `required:"true" split_words:"true"`
+	GitBranch             string        `default:"master" split_words:"true"`
+	GitCommitterEmail     string        `default:"kuberpult@freiheit.com" split_words:"true"`
+	GitCommitterName      string        `default:"kuberpult" split_words:"true"`
+	GitSshKey             string        `default:"/etc/ssh/identity" split_words:"true"`
+	GitSshKnownHosts      string        `default:"/etc/ssh/ssh_known_hosts" split_words:"true"`
+	GitNetworkTimeout     time.Duration `default:"1m" split_words:"true"`
+	GitWriteCommitData    bool          `default:"false" split_words:"true"`
+	PgpKeyRingPath        string        `split_words:"true"`
+	AzureEnableAuth       bool          `default:"false" split_words:"true"`
+	DexEnabled            bool          `default:"false" split_words:"true"`
+	DexRbacPolicyPath     string        `split_words:"true"`
+	EnableTracing         bool          `default:"false" split_words:"true"`
+	EnableMetrics         bool          `default:"false" split_words:"true"`
+	EnableEvents          bool          `default:"false" split_words:"true"`
+	DogstatsdAddr         string        `default:"127.0.0.1:8125" split_words:"true"`
+	EnableProfiling       bool          `default:"false" split_words:"true"`
+	DatadogApiKeyLocation string        `default:"" split_words:"true"`
+	EnableSqlite          bool          `default:"true" split_words:"true"`
+	DexMock               bool          `default:"false" split_words:"true"`
+	DexMockRole           string        `default:"Developer" split_words:"true"`
+	// ArgoCdServer             string        `default:"" split_words:"true"`
+	ArgoCdInsecure           bool   `default:"false" split_words:"true"`
```

Comment:
This should also be removed

Reply:
By this,do you mean all of those attributes?